roles/ceph-common/tasks/installs/install_rgw_on_debian.yml

```diff
@@ -104,7 +104,7 @@
 
 - name: install default httpd.conf
   template:
-    src: ../../templates/httpd.conf
+    src: "{{ playbook_dir }}/roles/ceph-common/templates/httpd.conf"
```

Comment:
Here are the three options I know of to specify a template in the templates directory. I would suggest the first one for looks.

- "httpd.conf"
- "{{ role_path }}/templates/httpd.conf"
- "{{ playbook_dir }}/roles/ceph-common/templates/httpd.conf"